t/20-function-status.t

```diff
@@ -59,7 +61,7 @@ my %init = (
     'job_name is correct');
   is ($d->command,
     'npg_status2file --id_run 1234 --status "run archived" --dir_out '
```

Comment:
Will you change this invocation first, or will that come after the other PR is merged?

Reply:
After the tracking pr is merged

Reply:
That's done, so you're free to update here too